int32 poseA
int32 poseB
---
bool state
